etc/config/rust.amazon.properties

```diff
@@ -944,10 +944,11 @@ tools.osacatrunk.type=postcompilation
 tools.osacatrunk.class=osaca-tool
 tools.osacatrunk.stdinHint=disabled
 
-tools.rustfmt1436.name=rustfmt (1.4.36)
-tools.rustfmt1436.exe=/opt/compiler-explorer/rustfmt-1.4.36/rustfmt
+tools.rustfmt1436.name=rustfmt (1.77.0)
```

Comment:
I didn't change the key, because if I remember correctly, that would break links using the tool?

Reply:
Oh yes; it would. For compilers we support "`.alias`" for this case...I don't know if our tools do.

Maybe pop a comment above explaining why the `id` is "wrong" and we can come back and look into it (unless you fancy quickly seeing if aliases are easy to add)

Reply:
Sure, I'll add a comment! I'll also bump to edition=2021 as default

Reply:
thanks @junlarsen !